src/HatsEligibilitiesChain.sol

```diff
@@ -153,28 +150,23 @@ contract HatsEligibilitiesChain is HatsEligibilityModule {
     }
 
     // check only standing for remaining modules, in case the wearer is eligible in a previous clause
-    if (clauseIndex < numClauses) {
-      while (clauseIndex < numClauses) {
-        length = _getArgUint256(104 + clauseIndex * 32);
-
-        for (uint256 j = 0; j < length;) {
-          module = _getArgAddress(clauseOffset + 20 * j);
-          (eligibleInModule, standingInModule) = HatsEligibilityModule(module).getWearerStatus(_wearer, _hatId);
+    while (clauseIndex < numClauses) {
+      length = _getArgUint256(104 + clauseIndex * 32);
+      nextClauseOffset += length * 20;
```

Comment:
can make this unchecked since the gas limit will be reached long before it ever overflows